modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i

```diff
@@ -211,4 +211,5 @@
   exodus = true
   print_linear_residuals = true
   print_perf_log = true
+  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
```

Comment:
We should be able to hide the variable by default if you want, it should be
as simple as setting some parameters on the objects within the action that
is creating them. Let me know if you want that and I can take a look.

On Thu, Mar 26, 2015 at 6:39 AM, Shane Stafford <notifications@github.com>
wrote:

> In
> modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i
> <https://github.com/idaholab/moose/pull/4851#discussion_r27212547>:
>
> > @@ -211,4 +211,5 @@
> >    exodus = true
> >    print_linear_residuals = true
> >    print_perf_log = true
> > +  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
>
> The variables don't necessarily need to be hidden; penetration and contact
> pressure are also automatic and do not get hidden. I just hid these to make
> the tests pass without changes to the gold files.
>
> I have no preference on hiding them by default, assuming that is possible
> to do in a clean way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4851/files#r27212547>.
>